spec/support/E2E_README.md

```diff
@@ -0,0 +1,62 @@
+# E2E Testing
+
+We have a few end-to-end tests located in [drivers/hmis/spec/system/hmis](drivers/hmis/spec/system/hmis).
+We are working on adding more coverage and running them in CI!
+Below are instructions for getting set up to run and write E2E tests locally.
+
+## Background and Testing Philosophy
+
+The goal of these tests is to catch regressions, especially in the UI. The tests use [Capybara](https://github.com/teamcapybara/capybara), which allows us to test our [frontend](https://github.com/greenriver/hmis-frontend/) using our existing factories to mock the backend models. This means these tests are testing both UI and API behavior.
```

Comment:
Should we add more detail here - for example talking about Cuprite and linking to the Evil Martians document?

Reply:
I think inline comments may be best for that kind of thing
* looks like we have that evilmartians article linked at the top of the e2e_tests file 
* for documentation of gem dependencies, I'd suggest annotating the Gemfile